dts/bindings/wifi/inventek,eswifi.yaml

```diff
@@ -0,0 +1,45 @@
+#
```

Comment:
Typo in commit "dts/bindings: Add bindings for es-WiFi module", s/byt/by/

Reply:
fixed